### PR TITLE
Compose Multiplatform setup

### PR DESCRIPTION
**Pull request by manuel-martos**

### CODE REVIEW

File: ios/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    cocoapods {
+        version = "1.0.0"
+        summary = "appyx-starter-kit iOS module"
+        ios.deploymentTarget = "16.4"
+        podfile = project.file("../iosApp/Podfile")
+        framework {
+            baseName = "ios"
+            isStatic = true
+        }
+    }
```
Comment (MikePT28) [inline ios/build.gradle.kts]: I tried running the branch and Cocoapods is complaining that is missing the fields for: homepage, author, and license.
I checked how it was on the demo project and it looks like this:
```
cocoapods {
    version = "1.0.0"
    summary = "appyx-starter-kit iOS module"
    homepage = "https://bumble-tech.github.io/appyx/"
    ios.deploymentTarget = "16.4"
    podfile = project.file("../iosApp/Podfile")
    framework {
        baseName = "ios"
        isStatic = true
    }
    license = "Apache License, Version 2.0"
    authors = "https://github.com/bumble-tech/"
}
```

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+[versions]
+accompanist = "0.28.0"
+agp = "8.1.1"
+androidCompileSdk = "34"
+androidMinSdk = "21"
+androidTargetSdk = "32"
+androidx-lifecycle = "2.6.1"
+androidx-navigation-compose = "2.5.1"
+appyx = "2.0.0-alpha10"
```
Comment (mapm14) [inline gradle/libs.versions.toml]:
```suggestion
appyx = "2.0.0-alpha09"
```

The latest version is 09, right? I can't download the 10

File: gradle/libs.versions.toml [hunk 3]
```diff
@@ -0,0 +1,110 @@
+# keep sorted please, use Edit -> Sort Lines
+
+[versions]
+accompanist = "0.28.0"
+agp = "8.1.1"
+androidCompileSdk = "34"
+androidMinSdk = "21"
+androidTargetSdk = "32"
+androidx-lifecycle = "2.6.1"
+androidx-navigation-compose = "2.5.1"
```
Comment (mapm14) [inline gradle/libs.versions.toml]: Is this necessary?

File: gradle/libs.versions.toml [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+kotlin-coroutines-rx2 = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-rx2", version.ref = "coroutines" }
+kotlinx-serialization-json = { module = "org.jetbrains.kotlinx:kotlinx-serialization-json", version.ref = "serialization-json" }
+
+rxjava2 = "io.reactivex.rxjava2:rxjava:2.2.21"
+rxjava3 = "io.reactivex.rxjava3:rxjava:3.1.5"
+rxandroid2 = "io.reactivex.rxjava2:rxandroid:2.1.1"
+rxandroid3 = "io.reactivex.rxjava3:rxandroid:3.0.2"
+rxrelay2 = "com.jakewharton.rxrelay2:rxrelay:2.1.1"
+rxrelay3 = "com.jakewharton.rxrelay3:rxrelay:3.0.1"
+
```
Comment (mapm14) [inline gradle/libs.versions.toml]: Is this needed?

**Review by mapm14 — APPROVED** [review]

File: ios/build.gradle.kts (hunk 1 above)
Comment (manuel-martos) [reply ios/build.gradle.kts]: Added missing values.

File: gradle/libs.versions.toml (hunk 2 above)
Comment (manuel-martos) [reply gradle/libs.versions.toml]: Good catch! Reverted!

**Pull request merged**